### PR TITLE
[13.0][ADD] stock_inventory_valuation_pivot

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT31612

This module extends the functionality of stock inventory valuation report to add a pivot view.

### CODE REVIEW

File: stock_inventory_valuation_pivot/views/stock_account_views.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            </field>
+        </field>
+    </record>
+
+    <record id="stock_valuation_layer_view_pivot" model="ir.ui.view">
+        <field name="name">stock.valuation.layer.pivot</field>
+        <field name="model">stock.valuation.layer</field>
+        <field name="arch" type="xml">
+            <pivot>
+                <field name="product_id" interval="month" type="row" />
```
Comment (sergio-teruel) [inline stock_inventory_valuation_pivot/views/stock_account_views.xml]: the field product_id is not a date field so you can not do that... change the this field by other of date type..

Selección_686

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

File: stock_inventory_valuation_pivot/views/stock_account_views.xml (hunk 1 above)
Comment (ernestotejeda) [reply stock_inventory_valuation_pivot/views/stock_account_views.xml]: well, the intention was to set product_id and not set `interval="month"`, so I'm going to remove `interval="month"`

**Comment by ernestotejeda** [comment]
@sergio-teruel changes done

**Comment by sergio-teruel** [comment]
Ok. Thanks.. when runbot is up I will test the module...

File: stock_inventory_valuation_pivot/views/stock_account_views.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+<odoo>
+    <record id="view_inventory_valuation_search" model="ir.ui.view">
+        <field name="model">stock.valuation.layer</field>
+        <field name="inherit_id" ref="stock_account.view_inventory_valuation_search" />
+        <field name="arch" type="xml">
+            <field name="product_tmpl_id" position="after">
+                <field
+                    string="Date Before"
+                    name="create_date"
+                    filter_domain="[('create_date', '&lt;=', self)]"
```
Comment (carlosdauden) [inline stock_inventory_valuation_pivot/views/stock_account_views.xml]: As the field is datetime, it is easier to filter records prior to a specific day at 00:00:00
```suggestion
                    filter_domain="[('create_date', '&lt;', self)]"
```

**Comment by ernestotejeda** [comment]
@carlosdauden changes done

**Comment by carlosdauden** [comment]
Runbot error installing setuptools, pip...
It did work OK yesterday, Any recent change in the repository?

**Review by carlosdauden — APPROVED** [review]

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**